client/src/panels/panel_declarations/services/services_digital_status.js

```diff
@@ -0,0 +1,151 @@
+import text from './services.yaml';
+import './services.scss';
+import { Service } from '../../../models/services.js';
+
+import {
+  create_text_maker_component,
+  declare_panel,
+  InfographicPanel,
+  NivoResponsiveHeatMap,
+  declarative_charts,
+  formats,
+} from "../shared.js";
+
+import {Fragment} from 'react';
+
+const { GraphLegend } = declarative_charts;
+
+const { text_maker, TM } = create_text_maker_component(text);
+
+const ServicesDigitalPanel = ({panel_args}) => {
+
+  const data_keys = ["account_reg_digital_status", "authentication_status", "application_digital_status", "decision_digital_status", "issuance_digital_status", "issue_res_digital_status"];
+
+  const heatmap_data = _.chain(panel_args.services)
+    .map(serv=>_.pick(serv,_.concat(["name"], data_keys)))
+    .map(serv=>_.each(serv, (value, key)=>{
+      if(value===null) serv[key] = NaN;
+      if(_.includes(data_keys, key)) serv[text_maker(key)] = serv[key];
```

Comment:
What is this, the Witcher?

That was a mutations joke.

Reply:
Ugh yeah, this is hacky, I should remake those objects. Nivo tends to be pretty free about presenting keys which is particularly problematic for us because of the need to put all text through text_maker.